e-shop-manager/src/test/java/it/unipd/mtss/business/BillImplTest.java

```diff
@@ -7,12 +7,38 @@
 
 import org.junit.Test;
 
-import it.unipd.mtss.business.exeption.EItemNotFoundException;
+import it.unipd.mtss.business.exception.BillException;
+import it.unipd.mtss.business.exception.EItemNotFoundException;
 import it.unipd.mtss.model.EItem;
 import it.unipd.mtss.model.EItemType;
+import it.unipd.mtss.model.User;
 
 public class BillImplTest {
 
+    @Test
+    public void testGetOrderPrice(){
+        BillImpl bill = new BillImpl();
+
+        List<EItem> items = new ArrayList<EItem>(List.of( 
+            new EItem(EItemType.Processor, "Processore1", 14.50),
+            new EItem(EItemType.Motherboard, "Scheda1", 12.50),
+            new EItem(EItemType.Processor, "Processore2", 10.50),
+            new EItem(EItemType.Processor, "Processore3", 6.50),
+            new EItem(EItemType.Keyboard, "Tastiera1", 16.50),
+            new EItem(EItemType.Mouse, "Mouse1", 3.50)
```

Comment:
Quando implemento il requisito 4 poi questo test fallisce perchè c'è lo stesso numero di mouse e tastiere, al limite aggiungi un mouse 
```suggestion
            new EItem(EItemType.Keyboard, "Tastiera1", 16.50),
            new EItem(EItemType.Mouse, "Mouse1", 2.50),
            new EItem(EItemType.Mouse, "Mouse2", 1)
```